ansible/roles/schulcloud-server-core/templates/common-cartridge-deployment.yml.j2

```
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: common-cartridge-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: common-cartridge
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: common-cartridge
+    app.kubernetes.io/component: files
```

This will be part from the Server so the component shoud be server and not files
```suggestion
    app.kubernetes.io/component: server
```

---

done ✅